src/hooks/reducerFunction.jsx

```diff
@@ -0,0 +1,27 @@
+export function reducerFunction(state, action) {
+    switch (action.type) {
+        case "name":
+            return { ...state, name: action.payload.target.value }
+        case "description":
+            return { ...state, description: action.payload.target.value }
+        case "time":
+            return { ...state, time: action.payload.target.value }
```

Comment:
`action.payload.target.value` can be destructured early in the code to avoid repetition.

Also, try to send the exact value in the first place.